formats/indexed.js

```diff
@@ -0,0 +1,45 @@
+const classic = require('./classic')
+const pify = require('promisify-4loc')
+const { QL0 } = require('ssb-subset-ql')
+
+module.exports = function () {
```

Comment:
All the other formats are objects, but this one is a function that returns an object. Could this be changed to object? I tripped over it when using this latest branch in replication-scheduler